src/Features/Lsif/GeneratorTest/OutputFormatTests.vb

```diff
@@ -89,11 +91,26 @@ Namespace Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.UnitTests
     ""type"": ""edge"",
     ""label"": ""contains""
   },
+  {
+    ""result"": [],
+    ""id"": 6,
+    ""type"": ""vertex"",
+    ""label"": ""foldingRangeResult""
+  },
+  {
+    ""outV"": 3,
+    ""inVs"": [
+      6
+    ],
+    ""id"": 7,
+    ""type"": ""edge"",
+    ""label"": ""textDocument/foldingRange""
+  },
```

Comment:
Verified this matches https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md#request-textdocumentfoldingrange